patches/MAGETWO-95591__customer_attributes_issue__2.2.6.patch

```diff
@@ -0,0 +1,1592 @@
+diff --git a/vendor/magento/module-catalog/Controller/Adminhtml/Product/Attribute/Save.php b/vendor/magento/module-catalog/Controller/Adminhtml/Product/Attribute/Save.php
```

Comment:
Change first line to be same as our other patches with parameters. Also drop index